frontend/src/scenes/data-warehouse/new/DataWarehouseTableForm.tsx

```diff
@@ -82,6 +82,15 @@ export function DatawarehouseTableForm(): JSX.Element {
                     />
                 </LemonField>
                 <div className="text-muted text-xs mb-4">
+                    {provider === 'google-cloud' && (
```

Comment:
Maybe just leave this notice for the error response? Seems like we could end up adding a wall of text here

Reply:
Ok, sounds good. 👍